src/main/cpp/Robot.cpp

```diff
@@ -121,16 +121,24 @@ void Robot::RobotPeriodic()
         feedback->UpdateFeedback();
     }
 
-    auto pigeon = RobotConfigMgr::GetInstance()->GetCurrentConfig()->GetPigeon(CanSensorUsage::CANSENSOR_USAGE::PIGEON_ROBOT_CENTER);
-    if (pigeon == nullptr)
+    auto currentConfig = RobotConfigMgr::GetInstance()->GetCurrentConfig();
+    if (currentConfig == nullptr)
     {
-        Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("DEUBGGING"), string("Pigeon Nullptr?"), "true");
+        Logger::GetLogger()->LogData(LOGGER_LEVEL::PRINT, string("DEUBGGING"), string("CurrentConfig Nullptr?"), "true");
     }
     else
```

Comment:
Why don't we just delete the whole else block?

This looks like debugging information and the swerve chassis already puts this and other related information out to the network tables.